libraries/AP_VisualOdom/AP_VisualOdom_Backend.cpp

```diff
@@ -71,6 +71,21 @@ void AP_VisualOdom_Backend::handle_vision_position_delta_msg(const mavlink_messa
                                   packet.confidence);
 }
 
+void AP_VisualOdom_Backend::handle_vision_speed_estimate(uint64_t remote_time_us, uint32_t time_ms, const Vector3f &vel)
+{
+    AP::ahrs().writeVisionSpeed(vel, time_ms);
+
+    // record time for health monitoring
+    _last_update_ms = AP_HAL::millis();
+
+    AP::logger().Write("VISS", "TimeUS,RemTimeUS,CTimeMS,VX,VY,VZ",
```

Comment:
I think these could arrive at a fairly high rate so we should move this logging to AP_Logger.  Right near the logging we have for VISP and VISO.  Because it's a velocity, maybe we should call it VISV?

Reply:
Thank you. I will change that